### PR TITLE
Cfp 2020

### PR DESCRIPTION
**Pull request by rachelkelly**

### CODE REVIEW

**Comment by altsalt** [comment]
I always liked the phrase that Vicky and others have uttered, "Submit early, submit often!" (maybe s/submit/propose/). Looks great!

**Comment by altsalt** [comment]
Oh, what do you think about having a plaintext url in there at the bottom? Just thinking about scrapers picking it up, or that final nudge to click. Dunno, just a thought.

File: _posts/2020-07-14-CFP-open.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+All members of the SeaGL Program Committee have agreed to operate according to the SeaGL [Code of Practice](https://seagl.org/news/2020/07/13/code_of_practice.html).
+
+
+### Speaker's Guide
+Here's some more detailed information about submitting your talk proposal to SeaGL 2020.
+
+#### Talk Format and Timing
+Talks will be presented remotely. Speakers may give their session live, or pre-record their session for the room moderator to play while they are in the chat session with the attendees.
+Talk length is 20 minutes, with another 10 minutes allowed for questions, for 30m total.
+We do not have longer time slots available at SeaGL 2020 because the online medium will make it more critical to present yourself concisely. Please do not propose talks that cannot be presented well within the allotted time. If you have a topic that absolutely needs more time, consider breaking your proposal into two talks: an introductory talk and a more "advanced" talk.
```
Comment (AndrewKvalheim) [inline _posts/2020-07-14-CFP-open.md:34]: These lines will be rendered as a single paragraph. Consider formatting them as a list ([`- `](https://kramdown.gettalong.org/quickref.html#lists)) or multiple paragraphs ([separate](https://kramdown.gettalong.org/quickref.html#paragraphs)).

File: _posts/2020-07-14-CFP-open.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+### Resources and Help
+Never presented at a conference or meetup before? Presented but still not feeling confident? It's OK: even the most experienced conference presenters aren't necessarily confident at this stuff.
+VM Brasseur's [Public Speaking repository](https://github.com/vmbrasseur/Public_Speaking#proposing-talks) has collected a lot of resources to help you level up your conference presenting.
+Please Note: We always end up being unable to invite a bunch of really great talks. It's not you, it's us: we have a limited number of speaking slots and need to draft and balance the conference program. If we turn you down this year, we encourage you to try again in the future (here and elsewhere).
+
+Don't let fear of rejection stop you from proposing. Please run your talk idea by us — we need you!
+
+Office Hours: Proposal Help and Mentoring
+Want to propose a talk but want feedback on your idea, proposal wording, talk title or just on how to deal with nerves? The Speaker Committee is running regular office hours online. We'll do everything we can to help you be successful with your proposal and presentation, from brainstorming to quick reviews to detailed walkthroughs.
```
Comment (AndrewKvalheim) [inline _posts/2020-07-14-CFP-open.md:84]: These lines will be rendered as a single paragraph. Consider formatting the first line as a heading ([`####`](https://kramdown.gettalong.org/quickref.html#headers)).

**Pull request merged**